CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing to kubevirt/kubevirtci
+
+Welcome! As stated in the [README](README.md) this repository contains code for the virtualized clusters used in testing KubeVirt.
+
+See [the KubeVirt contribution guide](https://github.com/kubevirt/kubevirt/blob/master/CONTRIBUTING.md) for general information about how to contribute.
+
+## Getting started with gocli
+
+Prerequisites:
+* python
+* Bazel
+
+Install Bazel according https://docs.bazel.build/versions/master/install.html
+Change dir to gocli folder:
+```
+cd cluster-provision/gocli
+```
+
+Using local gocli images durning development, and in order to test before publishing:
```

Comment:
/Charles Durning/during/

Reply:
> > I just thought this would work (i.e. clicking on "Details" to the right of the `dco` check), but it still doesn't as the link always goes against master :-)
> 
> after push it might work isnt it ? so its maybe if so
> https://github.com/kubevirt/kubevirtci/blob/master/CONTRIBUTING.md

You mean after merge, right? If so, then yes :-)